doc/API.md

```diff
@@ -0,0 +1,139 @@
+| ----------- | ------------ | ----------------------------------------------------------------------------------------------- |
+
+#### Response
+
+| Field         | Type                                      | Description                |
+| ------------- | ----------------------------------------- | -------------------------- |
+| `utxos`       | array of [Utxo resource](#utxo-resource)  | Unspent transaction output |
+
+
+### `getspendtx`
+
+The `getspendtx` RPC Command builds and returns the spend transaction given a
+set of utxos to spend.
+
+#### Request
+
+| Parameter   | Type                 | Description                        |
+| ----------- | -------------------- | ---------------------------------- |
+| `inputs`    | string array         | Utxo outpoints                     |
+| `outputs`   | map of string to int | Map of Bitcoin addresses to amount |
+| `feerate`   | int                  | Target feerate for the transaction |
+
+#### Response
+
+| Field      | Type                                                        | Description                    |
+| ---------- | ----------------------------------------------------------- | ------------------------------ |
+| `psbt`     | [Spend transaction resources](#spend_transaction_resources) | Spend transaction informations |
+
+
+##### Spend status
+
+Please note that this status refers only to the Spend transaction, with regarding to the signatures and the broadcast status.
+
+| Value           | Description                                                                                      |
+| --------------- | ------------------------------------------------------------------------------------------------ |
+| `non_final`     | The Spend transaction is not final, we are awaiting signatures either from managers or cosigners |
+| `pending`       | The transaction is not broadcasted to the Bitcoin network                                        |
+| `broadcasted`   | The Spend transaction has been broadcasted                                                       |
+| `confirmed`     | The Spend transaction has been confirmed in the blockchain                                       |
+| `deprecated`    | The Spend transaction cannot be used anymore, one of its input was spent by another transaction  |
+
+##### Spend transaction resources
+
+
+| Field               | Type         | Description                                |
+| ------------------- | ------------ | ------------------------------------------ |
+| `psbt`              | string       | Base64-encoded Spend transaction PSBT      |
+| `change_index`      | integer      | Index of the change output, might be null  |
+| `status`            | string       | [Spend status](#spend-status)              |
+
+### `broadcast`
+
+The `broadcast` RPC Command build and broadcast the transaction out of
+a signed psbt.
```

Comment:
How about an `updatetx` command that takes two parameters:
1. The PSBT
2. An optional boolean: "`broadcast`"

So this already supports multisigs

Reply:
I would prefer `updatepsbt`and we will need `listpsbts` and `deletepsbt`

Reply:
I changed for `updatespendtx`, listspendtxs, deletespendtx